### DISCUSSION TITLE
Autoscroll Going Off End of Screen

### DISCUSSION THREAD

**Discussion by wilgaboury** [haltu/muuri#547] [Q&A]
When I add dragAutoScroll and target just window or document.body, I get a strange effect where dragging an item to the right or bottom edge of the screen will cause the whole window to scroll out of bounds. This may be an error in my codebase but I wanted to ask if this is something that could be fixed proper Muuri configuration.

**Comment by niklasramo** [comment]
Yep, that's expected behaviour in some cases and you need to guard against it. It's explained [here](https://github.com/haltu/muuri#grid-option-dragautoscroll): 
> When you use this feature it is highly recommended that you create a fixed positioned element right under document.body and use that as the dragContainer for all the dragged items. If you don't do this and a dragged item's parent is auto-scrolled, the dragged item will potentially grow the scrolled element's scroll area to infinity unintentionally.
